README.md

```diff
@@ -62,12 +62,6 @@ Available options:
 $ nix-deploy --to parnell@remote-server --path $(nix-build --no-out-link --attr foo ~/Development/bar/release.nix)
 [+] Downloading: /etc/nix/signing-key.sec
 [+] Installing: /etc/nix/signing-key.sec
-[+] Downloading: /etc/nix/signing-key.pub
```

Comment:
I thought it did output these first two lines

Reply:
Oh oops, you're right! I guess I thought it was delegate's output for exchanging signing keys too.